### PR TITLE
package: langchain-voyageai

### PR DESCRIPTION
**Pull request by fzowl**
Thank you for contributing to LangChain!

**Description:** 
Introducing VoyageAI's new embedding and rerank models

- [ ] **Lint and test**: Run `make format`, `make lint` and `make test` from the root of the package(s) you've modified. See contribution guidelines for more: https://python.langchain.com/docs/contributing/

### CODE REVIEW

File: libs/partners/voyageai/langchain_voyageai/embeddings.py
Comment (YuxingZhang) [inline libs/partners/voyageai/langchain_voyageai/embeddings.py:66]: How did we get these numbers? @fzowl

**Review by YuxingZhang — APPROVED** [review]
LGTM besides my question. Please also wait for Hong to approve it before merging.

**Comment by Liuhong99** [comment]
lgtm. please go ahead and merge it. also please wait until both new rerankers and embeddings are officially online and then raise a pr to langchain

**Pull request merged**
